packages/slice-machine/scripts/migration/versions/0.3.0/index.ts

```diff
@@ -0,0 +1,37 @@
+import { FileSystem, Utils, Models } from "@slicemachine/core";
+
+import { Migration } from "../../migrate";
+
+// Migration to write previousNext or previousNuxt into the sm.json
+const migration: Migration = {
+  version: "0.3.0",
+  // eslint-disable-next-line @typescript-eslint/require-await
+  main: async function main({ cwd }): Promise<void> {
+    try {
+      const manifest = FileSystem.retrieveManifest(cwd);
+      if (!manifest.exists || !manifest.content) return;
+
+      const framework = Utils.Framework.defineFramework({ cwd });
+      let previousFramework = null;
```

Comment:
I think making a big condition
```suggestion
      if (framework === Models.Frameworks.next || framework === Models.Frameworks.nuxt) {}
```
would be better to avoid patching the manifest if we don't need to.

Reply:
and then 
```suggestion
      frameworkToSet = framework === Models.Frameworsk.next ? Models.Framework.previousNext : Models.Framework.previousNuxt
```

Reply:
I didi this to avoid writing big condition but as you prefer I can go back on it :)

Reply:
I change a little bit the code here, waiting for your feedback